docs/built_in_extensions.rst

```diff
@@ -0,0 +1,47 @@
+Built-in Extensions
+===================
+
+RPC
+---
+
+Nameko has a built-in implementation for RPC over AMQP. It includes the ``@rpc`` entrypoint, a proxy for services to talk to other services, and a standalone proxy that clients can use to make RPC calls to a cluster:
+
+.. literalinclude:: examples/rpc.py
+
+.. literalinclude:: examples/standalone_rpc.py
+
+Normal RPC calls block until the remote method completes, but proxies also have an "async" calling mode to background or parallelize RPC calls:
```

Comment:
Future is pretty standard terminology for what you get out of `async`, think it should be referenced.